README.md

```diff
@@ -21,7 +21,8 @@ Inspired by and ported from Christophe Riccio's ([@Groovounet](https://github.co
 |[sampler_object](http://webglsamples.org/WebGL2Samples/#sampler_filter)|:x:D3D not supported|:white_check_mark:| :x: D3D not supported|:white_check_mark:|
 |[sampler_wrap](http://webglsamples.org/WebGL2Samples/#sampler_filter)|:white_check_mark:|:white_check_mark:| :white_check_mark:|:white_check_mark:|
 |[sampler_filter](http://webglsamples.org/WebGL2Samples/#sampler_filter)|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|
-|[transform_feedback_interleaved](http://webglsamples.org/WebGL2Samples/#transform_feedback_interleaved)|:white_check_mark:|:grey_question:|:white_check_mark:|:grey_question:|
+|[transform_feedback_interleaved](http://webglsamples.org/WebGL2Samples/#transform_feedback_interleaved)|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|
+|[transform_feedback_interleaved](http://webglsamples.org/WebGL2Samples/#transform_feedback_interleaved)|:grey_question:|:white_check_mark:|:grey_question:|:x: not working|
```

Comment:
Testing on Windows:
Chrome 49: [.CommandBufferContext]GL ERROR :GL_INVALID_FRAMEBUFFER_OPERATION : glDrawArraysIntancedANGLE: framebuffer incomplete (clear)

Firefox 45: Error: WebGL: framebufferTextureLayer: attachment:: invalid enum value COLOR_ATTACHMENT0 fbo_rtt_texture_array.html:255:13
Error: WebGL: drawArraysInstanced: Incomplete framebuffer: Status 0x8cdd: Bad status according to the driver.